qemu/tests/cfg/ansible_test.cfg

```diff
@@ -0,0 +1,12 @@
+- ansible_test:
+    type = ansible_test
+    virt_test_type = qemu
+    playbook_repo = "https://github.com/ansible/test-playbooks.git"
+    # Top level playbook file
+    tl_playbook = "site.yml"
```

Comment:
Suggest using the full name `toplevel_playbook`

Reply:
Will update.